ArduCopter/Parameters.cpp

```diff
@@ -308,6 +308,13 @@ const AP_Param::Info Copter::var_info[] = {
     // @User: Advanced
     GSCALAR(flight_mode_chan, "FLTMODE_CH",         CH_MODE_DEFAULT),
 
+    // @Param: INITIAL_MODE
+    // @DisplayName: Initial flight mode
+    // @Description: This selects the mode to start in on boot. This is useful for when you want to start in AUTO mode on boot without a receiver.
+    // @Values: 0:Stabilize,1:Acro,2:AltHold,3:Auto,4:Guided,5:Loiter,6:RTL,7:Circle,9:Land,11:Drift,13:Sport,14:Flip,15:AutoTune,16:PosHold,17:Brake,18:Throw,19:Avoid_ADSB,20:Guided_NoGPS,21:Smart_RTL,22:FlowHold,23:Follow,24:ZigZag,25:SystemID,26:Heli_Autorotate
```

Comment:
where is standby mode? That is likely the most useful

Reply:
what do you call standby mode ?

Reply:
Ah, Standby mode is not actually a flight mode.  It's a modifier of any mode (I think).  For reference here is a list of modes: https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/mode.h#L15